src/plugins/legacy/variationalSegmentation/itkVariationalFunctionImageToImageFilter.h

```diff
@@ -86,9 +85,3 @@ class ITK_EXPORT VariationalFunctionImageToImageFilter:
 
 
 } // end namespace itk
-
-#ifndef ITK_MANUAL_INSTANTIATION
```

Comment:
It's done a lot in ITK. It may be a mistake to remove it indeed. I'm not used to includes at the bottom of files. `ITK_MANUAL_INSTANTIATION` is used also in `itkSphericalHarmonicITKToVTKFilter`.

Reply:
Done